plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -459,6 +461,44 @@ class RocketsPlugin::Impl
         });
     }
 
+    void _bindModelEndpoint(
+        const std::string& method, const std::string& key,
+        std::function<bool(std::string, ModelDescriptorPtr)> action)
+    {
+        _jsonrpcServer->bind(method, [&, key, action](
+                                         rockets::jsonrpc::Request request) {
+            ScopedCurrentClient scope(_currentClientID, request.clientID);
+
+            using namespace rapidjson;
+            Document document;
+            document.Parse(request.message.c_str());
+
+            if (!document.HasMember("id") || !document.HasMember(key.c_str()))
+                return Response::invalidParams();
+
+            const auto modelID = document["id"].GetInt();
+            auto model = _engine->getScene().getModel(modelID);
+            if (!model)
+            {
+                return Response{
+                    Response::Error{"Model not found", MODEL_NOT_FOUND}};
+            }
+
+            Document propertyDoc;
```

Comment:
A comment about what are you trying to do here makes reading code easier (and avoids guesses)

Reply:
done